Ai Pin/Views/Notes/NotesView.swift

```diff
@@ -19,8 +21,9 @@ struct MemoryView: View {
                                     .foregroundStyle(.red)
                             }
                         }
-                    Text(note.text)
+                    Text(.init(note.text))
```

Comment:
Is this so it will trigger markdown rendering?

Reply:
if so, it is preferred to use the proper named initializer.

Reply:
Yes! It was intended to trigger markdown. However, I'm noticing that leaving Text() by itself also triggers markdown